### PR TITLE
17.2.2 release notes

### PR DESCRIPTION
**Pull request by joaoandremartins**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
```
Comment (joaoandremartins) [inline CHANGELOG.md]: Is this required to stay?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+## [17.2.2]
+
+This is another maintenance release to address the invalid Java Runtime Environment (JRE) on 
+standard staging ([#1316](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1316))
```
Comment (etanshaul) [inline CHANGELOG.md]: Maybe just be a little more explicit: on staging an App Engine standard app

File: CHANGELOG.md (hunk 1 above)
Comment (etanshaul) [inline CHANGELOG.md]: I'm not sure what its for. I don't see it show up in the rendered notes. Maybe @patflynn knows?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
+## [17.2.2]
```
Comment (patflynn) [inline CHANGELOG.md]: shouldn't this be 17.2.2-2017?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+
+This is another maintenance release to address the invalid Java Runtime Environment (JRE) on 
+staging an App Engine standard app ([#1316](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1316))
+issue:
+
+> Unable to stage app: Cannot get the System Java Compiler. Please use a JDK, not a JRE.
```
Comment (patflynn) [inline CHANGELOG.md]: This doesn't seem to follow our standard format for changelog.

File: CHANGELOG.md (hunk 3 above)
Comment (etanshaul) [inline CHANGELOG.md]: we discussed offline that for the release notes we can just call it 17.2.2. you think we should be explicit that its 17.2.2_2017?

Comment (joaoandremartins) [inline CHANGELOG.md]: The "-2017" is just an artifact of us having two separate releases for each IJ product.
I wouldn't like to make it too public..

File: CHANGELOG.md (hunk 4 above)
Comment (joaoandremartins) [inline CHANGELOG.md]: I wasn't aware we had a changelog format.
The reason I added this bit here was so the issue becomes more discoverable. I can remove it if you want.

Comment (etanshaul) [inline CHANGELOG.md]: If you look at our other sections, we have a "Fixed" header where we list fixes.It has the issue referenced at the end.

**Comment by patflynn** [comment]
that's how it's displayed to users by jetbrains, so I'd prefer we kept the
full version string in sync.

**Comment by patflynn** [comment]
It's also fine to call out this particular issue in the summary at the top,
but also list it in the list of fixed issues.

**Comment by joaoandremartins** [comment]
Ah, hadn't understood what you meant.

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 5]
```diff
@@ -37,11 +37,24 @@
...
 
+      <h2>[17.2.2-2017]</h2>
+
+      <h3>Fixed</h3>
+
+      <ul>
+        <li>Invalid Java Runtime Environment (JRE) on staging an App Engine standard app
+        (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1316">#1316</a>):
+
+        <blockquote>Unable to stage app: Cannot get the System Java Compiler. Please use a JDK, not a JRE.</blockquote>
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: will this render properly? not sure if you can put a blockquote in an li like this (block element, inside of an inline element)?

File: CHANGELOG.md (hunk 1 above)
Comment (etanshaul) [inline CHANGELOG.md]: nm. li isn't inline. so its probably ok.

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml (hunk 5 above)
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: nm. li isn't inline. so its probably ok.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
+## [17.2.2-2017]
```
Comment (etanshaul) [inline CHANGELOG.md]: technically its 17.2.2_2017

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml (hunk 5 above)
Comment (joaoandremartins) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: I was actually waiting to do an alpha release to check that.
This bit will only render if the plugin is in a repository.

**Comment by etanshaul** [comment]
lgtm pending version display fix. I'm not sure about the 'unreleased' section - if its a convention or not. So I'm ok removing it since it doesn't break anything.

**Pull request merged**